sdk/typescript/src/types/objects.ts

```diff
@@ -149,7 +163,7 @@ export const SuiObjectData = object({
    * This can also be None if the struct type does not have Display defined
    * See more details in https://forums.sui.io/t/nft-object-display-proposal/4872
    */
-  display: optional(record(string(), string())),
+  display: optional(DisplayFieldsResponse),
```

Comment:
This is a breaking change, can you type this in a backwards compatible way, otherwise our products will stop working with testnet.